components/scripts/lib/gradle.sh

```diff
@@ -16,8 +16,8 @@ invoke_gradle() {
 
   if [ "$enable_ge" == "on" ]; then
     args+=("-Dcom.gradle.enterprise.build-validation.gradle.plugin-repository.url=https://plugins.gradle.org/m2")
-    args+=("-Dcom.gradle.enterprise.build-validation.gradle-enterprise.plugin.version=3.14.1")
-    args+=("-Dcom.gradle.enterprise.build-validation.ccud.plugin.version=1.12")
+    args+=("-Dcom.gradle.enterprise.build-validation.gradle-enterprise.plugin.version=3.14.1") # Max version for GE server 2023.2 -> See https://docs.gradle.com/enterprise/compatibility/#develocity_compatibility
```

Comment:
This comment is misleading to me since there is no max server version for the GE Gradle plugin 3.14.1. It is compatible with all current and future releases.
I suggest removing the comment along with the link to the compatibility matrix.

Reply:
3.14.1 is the max supported version for server 2023.2 though.

Reply:
Perhaps I should reword it to `Max version supported by GE server 2023.2 -> See https://docs.gradle.com/enterprise/compatibility/#develocity_compatibility`